packages/clerk-js/src/core/clerk.ts

```diff
@@ -1415,7 +1421,12 @@ export class Clerk implements ClerkInterface {
     }
     const provider = strategy.replace('web3_', '').replace('_signature', '') as Web3Provider;
     const identifier = await getWeb3Identifier({ provider });
-    const generateSignature = provider === 'metamask' ? generateSignatureWithMetamask : generateSignatureWithCoinbase;
+    const generateSignature =
+      provider === 'metamask'
+        ? generateSignatureWithMetamask
+        : provider === 'coinbase'
+          ? generateSignatureWithCoinbase
+          : generateSignatureWithCoinbaseWallet;
```

Comment:
In a follow up PR this will become simpler as we are dropping the `generateSignatureWithCoinbase` ?

Reply:
That's correct, on the next PR we are dropping the Coinbase provider in favor of the new Coinbase Wallet